packages/spark-core/src/plugins/credentials/credentials/browser.js

```diff
@@ -12,17 +12,18 @@ import {assign, clone, has, omit, pick} from 'lodash';
 import querystring from 'querystring';
 import url from 'url';
 import uuid from 'uuid';
-import Authorization from './authorization';
-import CredentialsBase from './credentials-base';
-import {persist, waitForValue} from '../../lib/storage';
+import Authorization from '../authorization';
+import common from './common';
+import {persist, waitForValue} from '../../../lib/storage';
+import SparkPlugin from '../../../lib/spark-plugin';
 
 /**
  * @private
  * @returns {undefined}
  */
 function noop() {/* eslint no-empty:[0] */}
 
-const Credentials = CredentialsBase.extend({
+const Credentials = SparkPlugin.extend(Object.assign(common, {
```

Comment:
Was this to make it play more nicely with the decorators using `@`, like `persist`?